site/views/_includes/landing/marquee.njk

```diff
@@ -15,9 +15,19 @@
   </div>
 
   <div class="marquee-illustration" role="presentation">
-    <div class="marquee-brand logo-animated">
-      {% include 'static/assets/' + landing.marquee.logo %}
-    </div>
+    {% set easterSecret = landing.marquee.easterSecret %}
+    {% if easterSecret %}
+      <a esl-d-marquee-secret-link
+         href="{{ easterSecret.url | url }}" rel="nofollow"
```

Comment:
We just don't want to pass SEO weight to this link, do we?

Reply:
At that moment - yes, exactly